keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/api/ImageSpec.kt

```diff
@@ -24,5 +24,5 @@ data class ImageSpec(
   override val maxDiffCount = 2
 
   @JsonIgnore
-  override val unhappyWaitTime = Duration.ZERO
+  override val unhappyWaitTime: Duration? = null
```

Comment:
Made this change in the relevant handler types. It looks like nothing ever uses any other value.